Composer deprecations fix

Ang3 opened the pull request:
Issue #18 

**Reason**
All classes in folder ```src/Ripcord/Documentor``` and ```src/Ripcord/Parsers``` have a wrong namespace definition following psr-4 specifications for classes autoloading.

**Resolution**
- Renamed all occurences of ```Documentator``` by ```Documentor```

**Comments**
I had to choice between words ```Documentor``` and ```Documentator```.

**Edit**
The word ```Documentor``` was choosen to avoid BC break.

Ang3 committed:
Composer deprecations fix

- Renamed all occurences of ```Documentor``` by ```Documentator```
- Renamed file ```src/Ripcord/Documentor.php``` by ```src/Ripcord/Documentator.php```
- Fixed file ```composer.json```

Ang3 committed:
phpstan configuration

Ang3 committed:
Deleted useless import

Ang3 commented:
How can I fix continuous integration checks?

DarkaOnLine commented on composer.json:
```diff
@@ -7,7 +7,7 @@
     "autoload": {
         "psr-4": {
-            "Ripcord\\": "src/Ripcord"
+            "Ripcord\\": "src/Ripcord/"
```
We should not need `/` at the end. Why this was added?

Ang3 replied:
Commit #ea4f923270b3c061bdc8412755173c042aee3eaf

DarkaOnLine commented on phpstan.neon:
```diff
@@ -0,0 +1,4 @@
+parameters:
+  checkMissingIterableValueType: false
+  paths:
+    - %currentWorkingDirectory%/src
```
Missing blank line at the end of file

Ang3 replied:
Commit #b3c674c273fcac162974ee8b5ee109e637760b08

Commit:
Renamed Documentator by Documentor

Renamed all occurences of ```Documentator``` by ```Documentor``` to avoid break compatibility.

Commit:
Deleted slash at end of psr-4 folder

Commit:
Added missing blank line

Commit:
Fixed Documentator class

- Renamed ```Documentator``` class to ```Documentor```

Ang3 commented:
It seems not possible to avoid all BC breaks due to the folder ```Documentator```.
Your code uses reflections so I didn't create fake folders and classes to extend right ones.
Maybe by using the parameter "classmap" of composer and remove psr-4 standard?
